react/features/lobby/middleware.ts

```diff
@@ -245,24 +208,45 @@ function _handleLobbyNotification(store: IStore) {
             dispatch(hideNotification(LOBBY_NOTIFICATION_ID));
             dispatch(rejectKnockingParticipant(firstParticipant.id));
         }) ];
-    } else {
+
+        if (showChat) {
+            customActionNameKey.splice(1, 0, 'lobby.chat');
```

Comment:
This reads really weird, I'd say it needs an explanatory comment.

Reply:
Like, this checks if lobby chat button is available and if so, it adds it to the customActionNameKey array?

Reply:
Yeah, an explanation of _what_ we are doing with that splice.